packages/eslint-plugin-pf-codemods/index.js

```diff
@@ -83,7 +84,8 @@ module.exports = {
         },
       },
       rules: Object.keys(rules).reduce((acc, rule) => {
-        acc[`pf-codemods/${rule}`] = "error";
+        const severity = rule.includes('warn') ? 'warn' : 'error';
+        acc[`@patternfly/pf-codemods/${rule}`] = severity;
```

Comment:
This allows us to set rules as warnings on the `eslint-plugin-pf-codemods` side of things.

Alternatively we could set it in the `eslintrc` file in `pf-codemods`, not sure which approach would be better and fine with either, but I lean *slightly* towards this so that it's a warning regardless of if the `pf-codemods` package is being used or the `eslint-plugin-pf-codemods`.

Reply:
I've since removed this as it seems some more discussion probably needs to happen around how we'd want this implemented, or if we even actually want it implemented.